crates/wasmtime/src/runtime/vm/libcalls.rs

```diff
@@ -1548,3 +1548,25 @@ fn cont_new(
         crate::vm::stack_switching::cont_new(store, instance, func, param_count, result_count)?;
     Ok(Some(AllocationSize(ans.cast::<u8>() as usize)))
 }
+
+#[cfg(feature = "gc")]
+unsafe fn get_instance_id(_store: &mut dyn VMStore, instance: Pin<&mut Instance>) -> u32 {
+    instance.id().as_u32()
+}
+
+#[cfg(feature = "gc")]
+unsafe fn throw_ref(
+    mut store: &mut dyn VMStore,
+    _instance: Pin<&mut Instance>,
+    exnref: u32,
+) -> Result<(), TrapReason> {
```

Comment:
Could `unsafe` from these two functions be dropped?

Reply:
Done!